network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,60 @@ These release notes track the development of the Network Observability Operator
+
+* link:https://access.redhat.com/errata/RHSA-2023:5379[ADD NEW ERRATA LINK Network Observability Operator 1.5.0]
+
+[id="network-observability-operator-1.5.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-dns-enhancements-1.5"]
+==== DNS tracking enhancements
+In 1.5, the TCP protocol is now supported in addition to UDP. New dashboards are also added to the *Overview* view of the Network Traffic page. For more information, see For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-dns-overview_nw-observe-network-traffic[Configuring DNS tracking] and xref:../network_observability/observing-network-traffic.
+
+[id="network-observability-RTT-1.5"]
+==== Round-trip time (RTT)
+In 1.5, you can use TCP handshake Round-Trip Time (RTT) captured from the `fentry/tcp_rcv_established` eBPF hookpoint to read SRTT to analyze network flows. In the *Overview*, *Network Traffic*, and *Topology* pages in web console, you can monitor network traffic and troubleshoot with RTT metrics, filtering, and edge labeling. For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-RTT-overview_nw-observe-network-traffic[RTT Overview] and xref:../network_observability/observing-network-traffic.adoc#network-observability-RTT_nw-observe-network-traffic[Working with RTT].
+
+[id="network-observability-metrics-dashboard-enhancements"]
+==== Metrics, dashboards, and alerts enhancements
+The Network Observability metrics dashboards in *Observe* → *Dashboards* → *NetObserv* has new metrics types you can use to create Prometheus alerts. The list items are now in the `includeList` whereas before they were in `ignoreTags`. For a complete list of these metrics, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-improved-lokistack-integration"]
+==== Improvements for Network Observability without Loki
+You can create Prometheus alerts for the *Netobserv* dashboard using DNS, Packet drop, and RTT metrics, even if you don't use Loki. In 1.4, these metrics were only available for querying and analysis in the *Network Traffic*, *Overview*, and *Topology* views, which are not available without Loki. For more information, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-enhanced-configuration-and-ui-1.5"]
+==== Notable enhancements 
+
+[discrete]
+[id="web-console-enhancements-1.5_{context}"]
+===== Web console enhancements:
+
+* The Differentiated Services Code Point (DSCP) field for egress QoS is now available in Network Observability. You can filter QoS DSCP in the web console *Network Traffic* page.
+* There are new panels added to the *Overview* view.
```

Comment:
PR is still pending, let's keep an eye on it but I feel this is not going to be part of 1.5:
https://issues.redhat.com/browse/NETOBSERV-1382
https://github.com/netobserv/network-observability-console-plugin/pull/431